File: precog/java/src/main/java/com/precog/api/dto/IngestResult.java
package com.precog.api.dto;

/**
 * Result of data ingestion
 * User: gabriel
 * Date: 10/26/12
 */
public class IngestResult {

    private int total;
    private int ingested;
    private int failed;
    private int skipped;
    private String[] errors;

    public int getTotal() {
        return total;
    }

    public void setTotal(int total) {
        this.total = total;
    }

    public int getIngested() {
        return ingested;
    }

    public void setIngested(int ingested) {
        this.ingested = ingested;
    }

    public int getFailed() {
        return failed;
    }

    public void setFailed(int failed) {
        this.failed = failed;
    }

    public int getSkipped() {
        return skipped;
    }

    public void setSkipped(int skipped) {
        this.skipped = skipped;
    }

    public String[] getErrors() {
        return errors;
    }

    public void setErrors(String[] errors) {
        this.errors = errors;
    }
}

File: precog/dotnet/PrecogClient/Client/Dto/IngestResult.cs
using System;

namespace Precog.Client.Dto
{
	public class IngestResult
	{
		public bool Completed { get; private set; }
		public decimal Total { get; set;}
    	public decimal Ingested { get; set;}
    	public decimal Failed { get; set;}
    	public decimal Skipped { get; set;}
    	public string[] Errors { get; set;}

		public IngestResult(bool completed= true)
		{
			this.Completed = completed;
		}
		
	}
}

